L1TriggerScouting/OnlineProcessing/plugins/BMTFStubMultiBxSelector.cc

```diff
@@ -0,0 +1,167 @@
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/EDPutToken.h"
+#include "FWCore/Utilities/interface/Exception.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+// L1 scouting
+#include "DataFormats/L1Scouting/interface/L1ScoutingBMTFStub.h"
+#include "DataFormats/L1Scouting/interface/OrbitCollection.h"
+
+#include <memory>
+#include <utility>
+#include <vector>
+#include <string>
+#include <numeric>
+#include <set>
+
+namespace l1ScoutingRun3 {
+  enum BMTFSelectorConditionType { Simple, Wheel };
+}
+
+class BMTFStubMultiBxSelector : public edm::stream::EDProducer<> {
+public:
+  explicit BMTFStubMultiBxSelector(const edm::ParameterSet&);
+  ~BMTFStubMultiBxSelector() override {}
+  static void fillDescriptions(edm::ConfigurationDescriptions&);
+
+  unsigned makeWheelPattern(const edm::Handle<OrbitCollection<l1ScoutingRun3::BMTFStub>>& stubs, unsigned bx);
+  bool windowHasCloseWheels(const std::vector<unsigned>& bxs);
+
+private:
+  void produce(edm::Event&, const edm::EventSetup&) override;
+
+  // Tokens for scouting data
+  edm::EDGetTokenT<OrbitCollection<l1ScoutingRun3::BMTFStub>> stubsTokenData_;
+
+  // Condition
+  l1ScoutingRun3::BMTFSelectorConditionType condition_;
+
+  // Selection thresholds
+  unsigned bxWindowLength_;
```

Comment:
Can you change `unsigned` to `unsigned int` to be clear in the data type throughout the PR please

Reply:
Done